partiql-planner/src/main/kotlin/org/partiql/planner/catalog/Catalog.kt

```diff
@@ -16,23 +18,19 @@ public interface Catalog {
 
     /**
      * Get a table by name.
+     *
+     * @param name  The case-sensitive [Table] name.
+     * @return The [Table] or null if not found.
      */
     public fun getTable(session: Session, name: Name): Table? = null
 
-    /**
-     * Get a table by identifier; note that identifiers may be case-insensitive.
-     */
-    public fun getTable(session: Session, identifier: Identifier): Table? = null
```

Comment:
Consider `getTable(session: Session, name: Name)` and `searchTable(session, identifier: Identifier)` or something similar.

We have no way of knowing which is the catalog/namespace, the table name, and the path expression — so we must do searching.